CHANGELOG.rst

```diff
@@ -14,6 +14,12 @@ Change Log
 Unreleased
 **********
 
+[5.0.0] - 2023-05-16
+********************
+Changed
+=======
+* **BREAKING CHANGE**: Removed deprecated ``signal`` argument from consumer initialization
```

Comment:
I wonder if it's worth setting a min-constraint on openedx-events 8.0.0 due to this

Reply:
Good call